bot/exts/info/doc/_cog.py

```diff
@@ -445,3 +468,7 @@ def cog_unload(self) -> None:
         self.inventory_scheduler.cancel_all()
         self.init_refresh_task.cancel()
         asyncio.create_task(self.item_fetcher.clear(), name="DocCog.item_fetcher unload clear")
+        
+        
+def predicate_emoji_reaction(ctx: Context, error_message: Message, reaction: Reaction, user: User):
```

Comment:
I'd honestly rather see this at the top before the cog rather than afterwards.

Reply:
Was just copying what was done in the [eval code](https://github.com/python-discord/bot/blob/main/bot/exts/utils/snekbox.py#L340-L342). Will move it up if you still think I should.

Reply:
In that case keep it like that for consistency.